soccer/MainWindow.cpp

```diff
@@ -318,6 +318,14 @@ void MainWindow::updateViews()
 
 	if (currentFrame)
 	{
+		if(_firstLogTimestamp == -1)
+			_firstLogTimestamp = currentFrame->timestamp();
+		auto gametime_ms = (currentFrame->timestamp()-_firstLogTimestamp)/1000;
```

Comment:
I agree, for something simple like ints, I think it's a lot easier to read with their actual types.  I went ahead and changed it.